templates/conditionorc-configmap.yaml

```diff
@@ -10,7 +10,7 @@ data:
     store_kind: serverservice
     # conditions accepted by this API
     conditions:
-      - kind: inventoryOutofband
+      - kind: inventory
```

Comment:
are we scrapping in-band/out-of-band? 🎉

Reply:
yep, the method of collection would be a parameter in the Condition payload 
https://github.com/metal-toolbox/mctl/blob/41f22410af443bab22cb36a48d2ba2df37447b44/cmd/collect/inventory.go#L48